components/Buttons/button.css

```diff
@@ -0,0 +1,62 @@
+:root{
+    --primary-color:black;  
+    --primary-btn-color: rgb(10, 177, 10);
+    --text-color:white;
+    --error-color:rgb(184, 18, 18);
+    --error-hover-color:red;
+        --btn-outline:black;
+        --btn-outline-primary:blue;
+        --btn-outline-success:rgb(0, 236, 0);
+        --btn-background-color:pink;
+}
+.btn{
```

Comment:
A `cursor : pointer` property on button, would increase the overall UX of the website.